sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/models/Paths108HwamOauth2ExchangePostRequestbodyContentApplicationXWwwFormUrlencodedSchema.java

```diff
@@ -11,48 +11,55 @@
 @Fluent
 public final class Paths108HwamOauth2ExchangePostRequestbodyContentApplicationXWwwFormUrlencodedSchema {
```

Comment:
nit: the name of this class and the one after this seem like bugs in naming

Reply:
Why so? The autorest generates that name because name is a function of those aspects, it just happens to become a super ugly name in this scenario :)